pkg/virt-handler/vm_test.go

```diff
@@ -655,6 +657,7 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(options.VirtualMachineSMBios.Product).To(Equal(virtconfig.SmbiosConfigDefaultProduct))
 				Expect(options.VirtualMachineSMBios.Manufacturer).To(Equal(virtconfig.SmbiosConfigDefaultManufacturer))
 			})
+			vmiInterface.EXPECT().Update(vmi).AnyTimes()
```

Comment:
Why is the update invoked in this test? IIUC, nothing is changed. Anyway, please be explicit and specify how many times it is invoked or nothing if it is invoked only once.

Reply:
Removed that now